index.html

```diff
@@ -50,9 +50,11 @@ <h2 class="pt-3 pb-1">A2 Retro Tech Mailing List:</h2>
     </div>
     <footer class="footer">
       <div class="container d-flex flex-row justify-content-center">
-        <img src="/img/floppy.svg" alt="icon depicting a floppy disk" class="d-inline-flex px-2 floppy">
-        <img src="/img/floppy.svg" alt="icon depicting a floppy disk" class="d-inline-flex px-2 floppy">
-        <img src="/img/floppy.svg" alt="icon depicting a floppy disk" class="d-inline-flex px-2 floppy">
+        <a href="https://www.flaticon.com/authors/creaticca-creative-agency" title="Icons made by Creaticca Creative Agency from www.flaticon.com">
```

Comment:
I think it's fair to give them attribution, but it's a bit weird to have these just link to their page. Is this their desired method of attribution, or could we use small text below the icons?